README.md

```diff
@@ -20,8 +20,19 @@ This project processes clinical data from https://clinicaltrials.gov/, specifica
 ## Set up
 
 ### For Windows Beginners
-
-*To be written*
+1)Open Windows PowerShell
```

Comment:
This is not how ordered lists are written in markdown. Please, revise https://www.markdownguide.org/basic-syntax/#lists-1